### DISCUSSION TITLE
#61 togETHer biweekly meeting

### DISCUSSION THREAD

**Discussion by bambalados** [tog-ETH-er/togETHer#46] [General]
Agenda:

- Community Updates
- Fireside Chat

会议主题：togETHer
会议时间： 21:00-22:00
重复周期：每两周 (周一)

点击链接入会：https://meet.google.com/fqc-kbzf-wxx

**Comment by Riely007** [comment]
### 以太坊翻译小组

- 日常翻译进行中，新增 Clients 和 Blog translations

**Comment by 0x-stan** [comment]
# Dapp-Learning

## repo update

6个PR #950 ～ #955

https://github.com/Dapp-Learning-DAO/Dapp-Learning/pulls

## sharing meeting

近两周举办了2场分享会

- 《Curve V2 CryptoSwap: whitepaper》by 0xstan
  - YouTube：https://www.youtube.com/watch?v=CKfoXO66wGc
  - Bilibili：https://www.bilibili.com/video/BV1xh411g7BP
 
- 《密码学系列课程5: Li17 两方签名与密钥刷新协议》by lynndell
  - Youtube:
    - p1 https://youtu.be/_cb0ujOS4qQ
    - p2 https://youtu.be/-qFdbe6lceU

  - Bilibili:
    - p1 https://www.bilibili.com/video/BV1Xa4y1N7cX/
    - p2 https://www.bilibili.com/video/BV1iM4y1C75z/

**Comment by liushooter** [comment]
## rebase

- web3 极客日报，最近 14 期 http://daily.rebase.network/
- 11 份招聘 https://github.com/rebase-network/who-is-hiring/issues

**Comment by Stamp9** [comment]
## 原语里弄
研究进行中。

**Comment by ethz42** [comment]
PlanckerDAO
--
1. AA社区筹备进行中, Aaron,昊昊,jason jiao,ben
2. 与mihai的Akasha团队召开合作会议
3. 公共物品wishpool、AA项目参与eth tokyo
4. eth tokyo side event: from ethereum to the decentralized world
4. 《收入邪恶曲线：思考公共物品融资优先的另一种方式》《网络国家之我见》中文翻译版本被merge进vitalik blog
5.  Remix China 正常推进中，翻译计划预算通过
6. LSD稳定币研究正常推进
7. 2024清迈devcon筹备小组启动

**Comment by bambalados** [comment]
CryptoTechNight:

规划下一次的CTN活动。

**Comment by 0x68616f79756e** [comment]
ETHPlanet:
筹备EDCON黑山活动
